cmake/Config.cmake.in

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+set(@PROJECT_NAME@_VERSION @PROJECT_VERSION@)
+
+@PACKAGE_INIT@
+
+# TODO(DF): add here any required find_dependency(), E.g.
+# include(CMakeFindDependencyMacro)
+# find_dependency(Imath @Imath_VERSION@ HINTS @Imath_DIR@)
+
+set_and_check (@PROJECT_NAME@_DIR "${PACKAGE_PREFIX_DIR}")
+set_and_check (@PROJECT_NAME@_INCLUDE_DIR "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_INCLUDEDIR@")
+set_and_check (@PROJECT_NAME@_LIBRARY_DIR "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_LIBDIR@")
+
+include ("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+
+check_required_components ("@PROJECT_NAME@")
```

Comment:
No blank line at EOF?

Reply:
Added blank lines and updated `.editorconfig` in f81c11d0